frontends/web/src/components/message/message.module.css

```diff
@@ -10,6 +10,7 @@
     margin-bottom: var(--spacing-default);
     overflow: hidden;
     padding: var(--spacing-default);
+    text-align: left;
```

Comment:
Does this now apply to all messages? Is this okay @thisconnect? This was added in response to https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/3158/#issuecomment-2647999195

Reply:
ack to set left align on all message components, until we have proper right-to-left langauge support (one day).